diff -ur ndprbrd-master/CMakeLists.txt ndprbrd-master.fix/CMakeLists.txt
--- ndprbrd-master/CMakeLists.txt	2017-08-07 04:48:21.000000000 -0700
+++ ndprbrd-master.fix/CMakeLists.txt	2017-10-31 22:03:10.607671756 -0700
@@ -21,4 +21,4 @@
 add_executable(ndprbrd ndprbrd.cpp)
 
 include(GNUInstallDirs)
-install(TARGETS ndprbrd RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
+install(TARGETS ndprbrd RUNTIME DESTINATION "${CMAKE_INSTALL_SBINDIR}")
